### ISSUE TITLE
The sylius_channel_countries is created into 2 migrations files 

### ISSUE THREAD

**Issue by kgonella** [Sylius/Sylius-Standard#399]
The following instructions to create the sylius_channel_countries table is created into 2 migrations files : 

- Version20200122082429.php
- Version20200301170604.php

```php
$this->addSql('CREATE TABLE sylius_channel_countries (channel_id INT NOT NULL, country_id INT NOT NULL, INDEX IDX_D96E51AE72F5A1AA (channel_id), INDEX IDX_D96E51AEF92F3E70 (country_id), PRIMARY KEY(channel_id, country_id)) DEFAULT CHARACTER SET UTF8 COLLATE `UTF8_unicode_ci` ENGINE = InnoDB');
$this->addSql('ALTER TABLE sylius_channel_countries ADD CONSTRAINT FK_D96E51AE72F5A1AA FOREIGN KEY (channel_id) REFERENCES sylius_channel (id) ON DELETE CASCADE');
$this->addSql('ALTER TABLE sylius_channel_countries ADD CONSTRAINT FK_D96E51AEF92F3E70 FOREIGN KEY (country_id) REFERENCES sylius_country (id) ON DELETE CASCADE');
```
That lead to an error when you setup your project

**Comment by kgonella** [comment]
It seems that the Version20200301170604.php contains the 3 lasts migrations data

**Comment by lchrusciel** [comment]
Hey Kevin,

thanks a lot for reporting. We fixed it in #406
